test/integration/highlight_editor_spec.mjs

```diff
@@ -331,4 +337,57 @@ describe("Highlight Editor", () => {
       );
     });
   });
+
+  describe("Text highlights aren't draggable", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait(
+        "tracemonkey.pdf",
+        ".annotationEditorLayer",
+        null,
+        null,
+        { highlightEditorColors: "red=#AB0000" }
+      );
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+    });
+
+    it("must check that a text highlight don't move when arrows are pressed", async () => {
+      await Promise.all(
+        pages.map(async ([browserName, page]) => {
+          await page.click("#editorHighlight");
+          await page.waitForSelector(".annotationEditorLayer.highlightEditing");
+
+          const rect = await getSpanRectFromText(page, 1, "Abstract");
+          const x = rect.x + rect.width / 2;
+          const y = rect.y + rect.height / 2;
+          await page.mouse.click(x, y, { count: 2 });
+
+          await page.waitForSelector(`${getEditorSelector(0)}`);
+          await page.waitForSelector(
+            `.page[data-page-number = "1"] svg.highlightOutline.selected`
+          );
+          await page.focus(getEditorSelector(0));
+
+          const xy = await getXY(page, getEditorSelector(0));
+          for (let i = 0; i < 100; i++) {
+            await page.keyboard.press("ArrowLeft");
+          }
```

Comment:
*Possibly a stupid question/idea, but:* Here, and similarly below, could we not make this part of the test a bit "quicker" by using just a few larger moves instead?
```suggestion
          for (let i = 0; i < 5; i++) {
            await kbBigMoveLeft(page);
          }
```